quality_estimation/mocha_eqqa_roberta_model.py

```diff
@@ -0,0 +1,132 @@
+from typing import Dict
+from overrides import overrides
+from allennlp.data import TextFieldTensors, Vocabulary
+from allennlp.models.model import Model
+from allennlp.modules import FeedForward
+from allennlp.nn import util
+from allennlp.training.metrics import MeanAbsoluteError, PearsonCorrelation
+
+import torch
+from torch.nn import MSELoss
+from transformers import AutoConfig, AutoModel, AutoTokenizer
+
+import logging
+
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+@Model.register("mocha_eqqa_roberta")
+class MochaQualityEstimator(Model):
+    def _init_layers(self):
```

Comment:
I'm curious why you need this initialization. Do you think Pytorch's default initialization is not good?